packages/frontend/src/components/Chores/ChoresLayout.js

```diff
@@ -3,8 +3,8 @@ import React, { useState, useEffect } from "react";
 import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
 import styles from "./ChoresStyle.module.css";
-import Chore from "./Chore";
 import { isAuthenticated } from "../auth";
+import mongoose from "mongoose";
```

Comment:
This line is why the CI/CD check failed. Why is it here? Do we need to be interacting with mongoose from the frontend?

Reply:
I suppose not, we did it to fix the types but I'll change it so it happens on the backend.